script/preview.js

```diff
@@ -84,6 +84,23 @@ if (process.env.SENTRY_DSN) {
   app.use(Raven.requestHandler());
 }
 
+/**
+ * Make the query params case-insensitive.
+ */
+app.use((req, res, next) => {
+  // eslint-disable-next-line fp/no-proxy
+  req.query = new Proxy(req.query, {
+    get: (target, name) =>
+      target[
+        Object.keys(target).find(
+          key => key.toLowerCase() === name.toLowerCase(),
+        )
+      ],
+  });
+
+  next();
```

Comment:
I'm not an express expert, but in the examples I've seen `next()` has been called conditionally if something didn't exist.  Since we don't have that condition, do we need this call?